db/seeds.rb

```diff
@@ -11,7 +11,32 @@
               posting_date: rand(1...10).days.ago,
               description: Faker::Lorem.sentence(3),
               location: "#{Faker::Address.city}, #{Faker::Address.state}",
-              company_id: rand(1...20),
+              company_id: rand(1...10),
               source: Faker::Name.name
             )
 end
+  puts "Jobs Created"
+
+10.times do
+  Company.create(
+                  name: "#{Faker::Company.name} #{Faker::Company.suffix}",
+                  url: Faker::Internet.url,
+                  location: "#{Faker::Address.city}, #{Faker::Address.state}",
+                )
+end
+  puts "Companies Created"
+
+20.times do
+  Tag.create(
+              name: Faker::Hacker.adjective,
+            )
+end
+  puts "Tags Created"
+
+100.times do
+  Jobtag.create(
+                 tag_id: rand(1...20),
+                 job_id: rand(1...20)
+               )
+end
+  puts "JobTags Created"
```

Comment:
Inconsistent indentation detected.<br>Do not write to stdout. Use Rails' logger if you want to log.